app/src/main/java/com/globant/practice/presentation/view/activity/HomeActivity.java

```diff
@@ -14,10 +25,24 @@
  */
 public class HomeActivity extends AppCompatActivity implements HomeView {
 
+    private RecyclerView listHomeRecyclerView;
+    private ProgressDialog fetchUserIndicator;
+    private SubscriberAdapter listHomeAdapter;
+    private HomeViewState homeViewState;
```

Comment:
The view must not manage a model attribute. The persistence must be managed in the presenter